config/webpackDevServer.config.js

```diff
@@ -26,8 +65,7 @@ module.exports = function(proxy, allowedHost) {
     // So we will disable the host check normally, but enable it if you have
     // specified the `proxy` setting. Finally, we let you override it if you
     // really know what you're doing with a special environment variable.
-    disableHostCheck:
-      !proxy || process.env.DANGEROUSLY_DISABLE_HOST_CHECK === 'true',
+    disableHostCheck: process.env.DANGEROUSLY_DISABLE_HOST_CHECK === 'true',
```

Comment:
Zouden we dit nog op false kunnen zetten? Of gaat er dan vanalles stuk? Is dit echt zo dangerous als het hier klinkt?

Reply:
Als iemand op het netwerk jouw dev server benadert, kan hij via de proxy settings bij de Lizard API met jouw credentials. Als we "host" aanpassen zodat hij alleen op localhost draait ipv alle interfaces dan is dat probleem weg.